spec/services/preserved_object_handler_spec.rb

```diff
@@ -67,9 +67,78 @@
   end
 
   describe '#check_existence' do
-    # FIXME: move the actual specs to a separate file?
-    it 'stubbed spec for now' do
-      skip
+    context "(calls create or confirm_version)" do
+      it 'calls #create_after_validation when the object does not exit' do
```

Comment:
as stated above, I don't think we want to test that it calls the method, as much as we want to test that the outcome is what we want, with the correct db changes and the correct result_array.  This sort of test is how we missed duplicate stuff.